ow_lander/scripts/trajectory_publisher.py

```diff
@@ -141,28 +116,30 @@ def talker(req):
     guarded_move_pub = rospy.Publisher('/guarded_move_result', GuardedMoveResult, queue_size=10)
 
     # Start subscriber for the joint_states
+    time.sleep(2) # subscribe right after coming from sleep .. this way we maintain values when moving downwards
```

Comment:
I don't understand this comment. Can you clarify it?

Reply:
```suggestion
    time.sleep(2) # wait for 2 seconds till the arm settles 
```